Tiltfile

```diff
@@ -0,0 +1,29 @@
+load('ext://restart_process', 'docker_build_with_restart')
```

Comment:
Should create the namespace `Build Failed: namespaces "wego-system" not found`.

Reply:
Nothing will work without installing first, so this is kind of moot.